### ISSUE TITLE
NullPointerException: instant

### ISSUE THREAD

**Issue by tonsky** [tonsky/grumpy#44]
```
Caused by: java.lang.NullPointerException: instant
        at java.base/java.util.Objects.requireNonNull(Objects.java:247) ~[na:na]
        at java.base/java.time.LocalDate.ofInstant(LocalDate.java:319) ~[na:na]
        at grumpy.time$format_date.invokeStatic(time.clj:15) ~[na:na]
        at grumpy.time$format_date.invoke(time.clj:14) ~[na:na]
        at grumpy.server$fn__19317.invokeStatic(server.clj:70) ~[na:na]
        at grumpy.server$fn__19317.invoke(server.clj:28) ~[na:na]
        at clojure.lang.AFn.applyToHelper(AFn.java:154) ~[grumpy.jar:na]
        at clojure.lang.AFn.applyTo(AFn.java:144) ~[grumpy.jar:na]
        at clojure.core$apply.invokeStatic(core.clj:665) ~[grumpy.jar:na]
        at clojure.core$apply.invoke(core.clj:660) ~[grumpy.jar:na]
        at rum.core$build_defc$fn__521.doInvoke(core.clj:137) ~[na:na]
        at clojure.lang.RestFn.invoke(RestFn.java:408) ~[grumpy.jar:na]
        at grumpy.server$fn__19361.invokeStatic(server.clj:88) ~[na:na]
        at grumpy.server$fn__19361.invoke(server.clj:86) ~[na:na]
        at clojure.lang.AFn.applyToHelper(AFn.java:154) ~[grumpy.jar:na]
        at clojure.lang.AFn.applyTo(AFn.java:144) ~[grumpy.jar:na]
        at clojure.core$apply.invokeStatic(core.clj:665) ~[grumpy.jar:na]
        at clojure.core$apply.invoke(core.clj:660) ~[grumpy.jar:na]
        at rum.core$build_defc$fn__521.doInvoke(core.clj:137) ~[na:na]
        at clojure.lang.RestFn.invoke(RestFn.java:408) ~[grumpy.jar:na]
        at grumpy.server$fn__19373.invokeStatic(server.clj:108) ~[na:na]
        at grumpy.server$fn__19373.invoke(server.clj:107) ~[na:na]
        at io.pedestal.interceptor$eval694$fn__695$fn__696.invoke(interceptor.clj:41) ~[na:na]
        at io.pedestal.interceptor.chain$try_f.invokeStatic(chain.clj:54) ~[na:na]
        ... 53 common frames omitted
```

**Comment by tonsky** [comment]
When requesting

```
/post/0Px1%E5%90%89%E6%9
E%97%E7%A6%8F%E5%BD%A9%E5%BF%AB%E4%B8%89_%E4%BF%A1%E8%AA%89%E7%BD%91%E6%8A%957Ukl
```

**Comment by tonsky** [comment]
Should be 404

**Comment by tonsky** [comment]
Fixed in faf8cd6b8a40816c6eb89f533655ec8277ca08ba
